v2/src/assets/icons/VShareToolsItem.vue

```diff
@@ -0,0 +1,266 @@
+<script setup>
```

Comment:
should this be in the icons folder?

Reply:
wtf... good catch.. must have been a copy paste mistake